electron-builder.json

```diff
@@ -0,0 +1,50 @@
+{
+  "productName": "Brim",
+  "asar": true,
+  "asarUnpack": [
+    "zdeps",
+    "LICENSE.txt",
+    "acknowledgments.txt"
+  ],
+  "directories": {
+    "buildResources": ".",
+    "output": "dist/installers"
+  },
+  "appId": "com.electron.brim",
+  "protocols": [
+    {
+      "name": "brim",
+      "schemes": [
+        "brim"
+      ]
+    }
+  ],
+  "afterSign": "electron-builder-notarize",
+  "mac": {
+    "entitlements": "./scripts/release/entitlements.mac.plist",
+    "icon": "./dist/static/AppIcon.icns",
+    "target": [
+      "dmg",
+      "zip"
+    ],
+    "binaries": [
+      "zdeps/zeek/lib/zeek/plugins/Corelight_CommunityID/lib/Corelight-CommunityID.darwin-x86_64.so"
+    ]
+  },
+  "nsis": {
+    "artifactName": "${productName}-Setup-${version}.${ext}",
+    "deleteAppDataOnUninstall": true
```

Comment:
I think we can revert this most recent commit. Just to write down the wider context for the PR history, I'd made this observation on Slack to @mason-fish:

> I've not found this to cause any problems, but just sharing an observation: With the new builds, after doing an uninstall on Windows, a directory `AppData\Local\brim-updater` is left behind with the single file `installer.exe` in it.

He added this `"deleteAppDataOnUninstall": true` in response to that. But I just confirmed in my testing fork that this setting does _not_ trigger the removal of that file/directory. Rather, I saw it triggered the removal of the `AppData\Local\Brim` directory, which is where the user's `appData.json` lives along with the `data` directory that holds all the ZNG for the Spaces and so forth. Our older builds left that `Roaming` directory intact and I think we might as well continue to behave this way since it allows a user to uninstall/reinstall and still see all their data (I just confirmed that works with the last GA release, tag `v0.24.0`). If we ever wanted to get fancy perhaps one day we could find a way to prompt the user and delete its contents on-demand during uninstall if that's what they want. But for now to keep things simple I'd say we should revert that commit so the `Roaming` dir is left behind after uninstall.

And back to the original point, I'm not too concerned about that `brim-updater` directory. The old system also left behind droppings after uninstall in the form of an `AppData\Local\Brim` directory that contained a file called `.dead` and another file `Update.exe`. So, there's precedent here without ever having heard of it causing a problem. I just like to document a sense of what's typical so that way I know to watch out for variations from this as a sign that something's wrong.